widget/playground/src/components/Slider/Slider.tsx

```diff
@@ -19,8 +20,21 @@ function Slider(props: PropTypes) {
     variant = 'custom',
     min,
     max,
+    id,
   } = props;
 
+  const progressScript = () => {
+    const sliderEl = document.querySelector(`#${id}`) as HTMLInputElement;
+
+    const sliderValue = parseInt(sliderEl.value);
+    const maxValue = max || 100;
+    const m = sliderValue * (100 / (maxValue as number));
```

Comment:
I think the variables' name can be better in this file.

Reply:
fixed